### PR TITLE
Add GenTX and Peer moon

### PR DESCRIPTION
**Pull request by Damian00000**
Moniker: moon
Twitter page: https://twitter.com/Denis24534
Team Discord: Denis#6284
Neutron Valoper: neutronvaloper1ha47m8l33c2esa8hzgwwgc24xaj64sqkr2507w

### CODE REVIEW

File: quark/gentxs/gentx.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"moon","identity":"31A1172488DADFDA","website":"","security_contact":"","details":"professional validator"},"commission":{"rate":"0.050000000000000000","max_rate":"0.050000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1ha47m8l33c2esa8hzgwwgc24xaj64sqkehdkc2","validator_address":"neutronvaloper1ha47m8l33c2esa8hzgwwgc24xaj64sqkr2507w","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"7VoMb2XoU/9yCdxOnFtIKoIDYksduLzc9WV28KQWmW8="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"b2f48da7421dd571ffa25e253fd0518dc2315d1c@45.142.214.235:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AwMIuXN+XzoJL7sDbUmomHxHwjyfi1XvTOKmbXEHx2+e"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["kKUyn4jdHn6QURa6Ul5HBhjWGhHy5DJeKkqJsFGB1GltPLpAXTz7UnpsQtNAt5DTTdD3rn0SLOifr6MrPPg2TQ=="]}
```
Comment (albertandrejev) [inline quark/gentxs/gentx.json:1]: Please update `max_change_rate` in accordance with the documentation.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/gentx.json (hunk 1 above)
Comment (Damian00000) [reply quark/gentxs/gentx.json]: Updated

**Comment by Damian00000** [comment]
Moniker: moon
Twitter page: https://twitter.com/Denis24534
Team Discord: Denis#6284
Neutron Valoper: neutronvaloper1ha47m8l33c2esa8hzgwwgc24xaj64sqkr2507w

**Review by albertandrejev — APPROVED** [review]

**Comment by Medniyy** [comment]
Unfortunately, we can't give you the validator role because you've not entered our Discord! 
This is the link: https://discord.gg/r82yeMu9Rf
Please enter and ping soi2studio#1446 with the link of this PR to receive your role.

**Pull request merged**